prime-router/docs/observability/azure-events.md

```diff
@@ -202,55 +93,43 @@ customEvents
 ### Get reportable conditions count for all reports sent to Report Stream
 ```
 customEvents
-| where name == "ReportAcceptedEvent"
-| extend observations = parse_json(tostring(customDimensions.observations))
-| mv-expand observations
-| extend conditions = parse_json(tostring(observations.conditions))
-| mv-expand conditions
-| extend conditionDisplay = tostring(conditions.display)
+| where name == "ITEM_ACCEPTED"
+| extend conditionDisplay = tostring(parse_json(tostring(parse_json(tostring(parse_json(tostring(parse_json(tostring(parse_json(tostring(customDimensions.params)).bundleDigest)).observationSummaries))[0].testSummary))[0].conditions))[0].display)
```

Comment:
there has to be a better way 😆 

If it works, it works though.

Reply:
I just used the UI in azure to extend on that field and that's what it gave me. I agree it looks kinda messy though